cross-chain/solana/scripts/deploy.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -eo pipefail
+
+# Setting env variables in the current bash shell
+source solana.env
+
+[ -z "$CLUSTER" ] && {
+  echo "'--cluster' option not provided" >&2
+  help
+  exit 1
+}
+
+[ -z "$AUTHORITY" ] && {
+  echo "'AUTHORITY' env var is not set" >&2
+  exit 1
+}
+
+echo "Building workspace for cluster: $CLUSTER ..."
+anchor build --provider.cluster $CLUSTER
```

Comment:
Use `NETWORK=solana-devnet make build` instead. This will create a directory called `artifacts-testnet` which will have the compiled BPF that you will deploy.

See Makefile for more info.

Reply:
I forgot about your env file. You can export NETWORK in your env file, too.

Reply:
Sure, done.